doc/source/templates/configs/anyscale_cluster_env.yaml

```diff
@@ -1 +1,7 @@
-docker_image: anyscale/ray-ml:latest-py39
+# you can define a custom byod definition e.g.
+# byod:
+#   docker_image: anyscale/ray-ml:latest-py39
+#   ray_version: nightly
+# or define a build_id for existing images, e.g. for nightly:
+# build_id: "anyscaleray-mlnightly-py39-gpu"
+build_id: "anyscaleray-ml231-py39-gpu"
```

Comment:
Is it possible to use a latest alias here so that it doesn't need to be updated with every release?

Reply:
As far as I know we don't support that from the anyscale side